war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -128,7 +131,7 @@ var createPluginSetupWizard = function() {
 
 	// Instantiate the wizard panel
 	var $wizard = $(pluginSetupWizard());
-	$wizard.appendTo('body');
+	$wizard.appendTo(appendTo);
```

Comment:
What is `appendTo`? I see it passed in. The name made me think it was a callback of some sort, but then it looks like it's used as a jquery object.